src/components/canvas-tools/RedoContainer.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-btn
+    class="undoBtn"
+    icon
+    elevation="2"
+    @click="onButtonClickHandler"
```

Comment:
event handlers should be at the bottom of the list of props. 
value should be at the top, followed by id then the rest of the props.